src/extensions/lens-main-extension.ts

```diff
@@ -1,7 +1,12 @@
 import type { MenuRegistration } from "./registries/menu-registry";
 import { observable } from "mobx";
 import { LensExtension } from "./lens-extension"
+import { WindowManager } from "../main/window-manager";
 
 export class LensMainExtension extends LensExtension {
   @observable.shallow appMenus: MenuRegistration[] = []
+
+  async navigate(location: string, frameId?: number) {
```

Comment:
Should we add similar method to `lens-renderer-extension.ts`?

Reply:
Yes, done. I still left `Navigation.navigate` because that's handy in React components.